python/lsst/drp/tasks/gbdesAstrometricFit.py

```diff
@@ -1233,3 +1263,47 @@ def _make_outputs(self, wcsf, visitSummaryTables, exposureInfo):
             catalogs[visit] = catalog
 
         return catalogs
+
+    def _output_model_params(self, wcsf):
+        """Get the WCS model parameters and covariance and convert to a
+        dictionary that will be readable as a pandas dataframe.
+
+        Parameters
+        ----------
+        wcsf : `wcsfit.WCSFit`
+            WCSFit object, assumed to have fit model.
+
+        Returns
+        -------
+        modelParams : `dict`
+            Parameters and covariance of the best-fit WCS model.
+        """
+        modelParamDict = wcsf.mapCollection.getParamDict()
+        modelCovariance = wcsf.getModelCovariance()
+
+        modelParams = {"mapName": [], "coordinate": [], "parameter": [], "coefficientNumber": []}
```

Comment:
It doesn't particularly matter with a short list but this could be a dict comp or defaultdict.